Program.cs

```diff
@@ -3,18 +3,29 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to Swap a given number
+// User gives two numbers to be swapped and the swapped numbers are displayed.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Swap numbers</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method gets the numbers from the user and displays the swapped numbers</summary>
+      static void Main () {
+         Console.WriteLine ("Enter two numbers to be swapped: ");
+         if (!double.TryParse (Console.ReadLine (), out double firstNumber)) Console.WriteLine ("Invalid Input.");
+         else {
+            if (!double.TryParse (Console.ReadLine (), out double secondNumber)) Console.WriteLine ("Invalid Input.");
+            else Console.WriteLine (Swap (firstNumber, secondNumber));

```

Comment:
Instead of just printing the numbers in brackets, display the output in a way that shows the previous values of a and b and the current values after swapping. See a sample below.
Value of a = 20, b = 40
After swapping, a = 40, b = 20